src/vs/workbench/contrib/debug/browser/baseDebugView.ts

```diff
@@ -50,7 +51,7 @@ export function replaceWhitespace(value: string): string {
 	return value.replace(/[\n\r\t]/g, char => map[char]);
 }
 
-export function renderExpressionValue(expressionOrValue: IExpression | string, container: HTMLElement, options: IRenderValueOptions): void {
+export function renderExpressionValue(expressionOrValue: IExpression | string, container: HTMLElement, linkDetector: LinkDetector, options: IRenderValueOptions): void {
```

Comment:
I would prefer if the linkDetector is part of the `IRenderValueOptions` and that it is optional. So we giva a chance for some clients to decide to not render links